src/main/java/com/netflix/msl/entityauth/X509AuthenticationData.java

```diff
@@ -42,7 +45,7 @@
  * <p>
  * {@code {
  *   "#mandatory" : [ "x509certificate" ],
- *   "x509certificate" : "base64"
+ *   "x509certificate" : "base64 DER-encoded certificate"
```

Comment:
Base64-encoding of DER is PEM. Will it accept the START and END string markers? I think it will.